packages/components/pie-tag/src/index.ts

```diff
@@ -11,10 +13,38 @@ const componentSelector = 'pie-tag';
 
 /**
  * @tagname pie-tag
+ * @slot icon - The icon slot
+ * @slot - Default slot
  */
 export class PieTag extends LitElement implements TagProps {
+    @property({ type: String })
+    @validPropertyValues(componentSelector, variants, 'neutral')
+    public variant: TagProps['variant'] = 'neutral';
+
+    @property({ type: String })
+    @validPropertyValues(componentSelector, sizes, 'large')
+    public size : TagProps['size'] = 'large';
+
+    @property({ type: Boolean })
+    public isStrong = false;
+
     render () {
-        return html`<h1 data-test-id="pie-tag">Hello world!</h1>`;
+        const {
+            variant,
+            size,
+            isStrong,
+        } = this;
+        return html`
+            <div
+                class="c-tag"
+                variant=${variant}
+                size=${size}
+                ?isStrong=${isStrong}
+                data-test-id="pie-tag"
+            >
+                ${size === 'large' ? html`<slot name="icon"></slot>` : nothing}
```

Comment:
Do you need a check to make sure that the icon is enabled? I'd assume you can show a large tag element without an icon too so here i think if size was large but we didn't want an icon it would render something here?

Reply:
@kevinrodrigues As it's a slot, if they don't want an icon then they just don't pass one in 😄

Reply:
🤦 Yes you're right.